app/(dashboard)/instances/components/InstanceForm.tsx

```
@@ -127,269 +127,291 @@ const InstanceForm = ({
+            const result = filterSchema.find((schemaParam) => {
+              return schemaParam.key === key;
+            });
+
+            switch (result?.type?.toLowerCase()) {
+              case "number":
+                if (data.requestParams[key] === "") break;
+                data.requestParams[key] = Number(data.requestParams[key]);
+                break;
+              case "float64":
+                if (data.requestParams[key] === "") break;
+                const output = Number(data.requestParams[key]);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)